templates/livecoms.cls

```diff
@@ -213,9 +213,9 @@
 % nothing
 \fi
 
-\if@comparison
+\if@software
 \documenttypetrue
-\newcommand{\documenttype}{Molecular Simulation Comparison}
+\newcommand{\documenttype}{Software Analyses Paper}
```

Comment:
Maybe it doesn't matter for this, but do we want Software Analyses [or Analysis] papers?  I prefer analysis.

Reply:
Yeah, that was the question I proposed on Issue #76 associated with this PR